test/parallel/test-http-response-add-header-after-sent.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  assert.doesNotThrow(() => {
+    res.setHeader('header1', 1);
+  });
+  res.write('abc')
+  assert.throws(() => {
+    res.setHeader('header2', 2);
+  }, Error, 'setHeader after write should throw');
+  res.end()
+});
+
+server.listen(common.PORT, () => {
+  http.get({port: server.address().port}, () => {
+    server.close()
+  });
```

Comment:
Ok, I will fix that. Just a question about it... In the [guide](https://github.com/nodejs/node/blob/master/doc/guides/writing_tests.md) for how to write tests it is mentioned that you should "always use common.PORT instead of an arbitrary value as it allows to run tests in parallel safely". It doesn't mention that you can use port 0 as well. If it is preferable to use port 0 than maybe the guide should be updated? Or did I misunderstand the guide?